NUITizenGallery/Examples/AnimatedVectorImageViewTest/AnimatedVectorImageViewTest.xaml.cs

```diff
@@ -0,0 +1,123 @@
+﻿/*
+ * Copyright(c) 2021 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+using Tizen.NUI;
+using Tizen.NUI.BaseComponents;
+using Tizen.NUI.Components;
+
+namespace NUITizenGallery
+{
+    using tlog = Tizen.Log;
+
+    public partial class AnimatedVectorImageViewTestPage : ContentPage
+    {
+        private const string tag = "NUITEST";
+        private int toggle, toggle2, toggle3;
+
+        public AnimatedVectorImageViewTestPage()
+        {
+            InitializeComponent();
+            tlog.Error(tag, $"aviv TotalFrame={aviv.TotalFrame}");
+            ChangeCurrFrameBtn.Clicked += ChangeCurrFrameBtnClicked;
```

Comment:
callback can be added in XAML also but this is okay I think.

Reply:
thank you for review. I will follow your advice from next time~